demo/Render.hpp

```diff
@@ -0,0 +1,841 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+#ifndef PEGASUS_RENDER_HPP
+#define PEGASUS_RENDER_HPP
+
+#include <glbinding/gl44ext/gl.h>
+#define GLFW_INCLUDE_NONE
+#include <GLFW/glfw3.h>
+#include <glm/ext.hpp>
+
+#include <vector>
+
+using namespace gl;
```

Comment:
Please avoid such blanket use of `using namespace` in global scope (or any scope) in header files

Reply:
I'm not planning to link any other OpenGL extension libraries and I am not planning to redefine any of the OpenGL functions in my code. OpenGL itself has every function declared with "gl" prefix an additional guaranty to avoid name collisions. So I don't see a problem with this one. I can move it in the specific header file though.

Reply:
Sure, it's just a general rule to never use `using namespace` in headers